src/scripts/common.js

```diff
@@ -143,16 +145,28 @@ window.togglbutton = {
                     clearTimeout(debouncer);
                   }
                   debouncer = setTimeout(function () {
-                    togglbutton.renderTo(selector, renderer);
+                    if (opts.iframeSelector) {
+                      togglbutton.renderToIframe(opts.iframeSelector, selector, renderer);
+                    } else {
+                      togglbutton.renderTo(selector, renderer);
+                    }
```

Comment:
I honestly would do one `togglbutton.render(selector, renderer, context: 'iframe' | 'document', optionaliframeSelector)` but I don't know which approach is better 🙈